src/os/FileStore.cc

```diff
@@ -530,9 +529,11 @@ FileStore::FileStore(const std::string &base, const std::string &jdev, osflagbit
   stop(false), sync_thread(this),
   fdcache(g_ceph_context),
   wbthrottle(g_ceph_context),
+  next_osr_id(0),
   throttle_ops(g_ceph_context, "filestore_ops",g_conf->filestore_queue_max_ops),
   throttle_bytes(g_ceph_context, "filestore_bytes",g_conf->filestore_queue_max_bytes),
-  op_finisher(g_ceph_context),
+  ondisk_finisher_num(g_conf->filestore_ondisk_finisher_threads),
+  apply_finisher_num(g_conf->filestore_apply_finisher_threads),
```

Comment:
prefix these with m_ and put them next to all the other tunables, a few lines down?